plugins/inputs/riak/riak.go

```diff
@@ -36,52 +36,105 @@ func NewRiak() *Riak {
 
```

Comment:
The plugin doesn't check the server version anywhere. Are these new fields always available on riak, even on older versions? Similar plugins check server version and only request fields that are valid on that version.

Reply:
Some of these metrics are only available in the latest versions. I will look into how other plugins handle this. In this first attempt, I was hoping to get the plugin to "ignore" metrics that aren't available in the response from the server (eg. because they are not supported by the server version).